examples/models/unet.py

```diff
@@ -0,0 +1,148 @@
+import torch.nn as nn
+from models.modules.layers_3d import (SparseConvBlock, SparseDeConvBlock,
+                                      SparseResBlock)
+
+import torchsparse
+import torchsparse.nn as spnn
+
+__all__ = ['SparseResUNet']
+
+
+class SparseResUNet(nn.Module):
+
+    def __init__(self, **kwargs):
+        super().__init__()
+
+        cr = kwargs.get('cr', 1.0)
+        cs = [32, 32, 64, 128, 256, 256, 128, 96, 96]
+        cs = [int(cr * x) for x in cs]
+
+        self.stem = nn.Sequential(
+            spnn.Conv3d(4, cs[0], kernel_size=3, stride=1),
+            spnn.BatchNorm(cs[0]), spnn.ReLU(True),
+            spnn.Conv3d(cs[0], cs[0], kernel_size=3, stride=1),
+            spnn.BatchNorm(cs[0]), spnn.ReLU(True))
```

Comment:
Nit: add a `,` after `spnn.ReLU(True)` so that YAPF will format it slightly better.